proj_circle.yml

```diff
@@ -4,9 +4,13 @@ machine:  # remeber to update those!
   node:
     version: 6.1.0
 
+dependencies:
+  pre:
+    - curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
```

Comment:
Maybe we should add a comment explaining why we are doing this since this will end up in the final project and users of the boilerplate will be confused about what is this.